launch/crp_launcher/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ endif()
 find_package(ament_cmake REQUIRED)
 
 install(DIRECTORY
+  config
```

Comment:
The config in crp_launcher doesn't exist anymore in main. Merging with main should delete this.